### ISSUE TITLE
Dota 2 crashes after the launch

### ISSUE THREAD

**Issue by darkbaboon** [ValveSoftware/Dota-2#2221]
#### Your system information

* System information from steam (`Steam` -> `Help` -> `System Information`) in a [gist](https://gist.github.com/darkbaboon/7e7976069632ad09002895784f6aaa21.js): 
* Have you checked for system updates?: Yes
* Are you using the latest stable video driver available for your system? Yes
* Have you verified the game files?: Yes

#### Please describe your issue in as much detail as possible:
Launch the game, crash a few seconds later and the button Play turns green again.
It seems a library is missing since the last update : librenderdoc.so
Please, if it helps, see the log from Steam during the launch : [Github Gist](https://gist.github.com/darkbaboon/a0dbb55d5f3639b40ead73c4ec8ff139).

#### Steps for reproducing this issue:

1. Launch the game.
2. A few seconds later the launch stopped and the button Play to launch the game turns green again.
3. I use Vulkan, but the library librenderdoc.so which seems to be related to OpenGL seems to be necessary to launch the game.

**Comment by kisak-valve** [comment]
Hello @darkbaboon, librenderdoc missing is expected and not fatal. That is a red herring.

Can you check if any Vulkan-based game works? https://gist.github.com/darkbaboon/7e7976069632ad09002895784f6aaa21#file-gistfile1-txt-L378-L394 in your system information doesn't look healthy, but a timeout might also be a false indicator from the system being busy with something else.

**Comment by darkbaboon** [comment]
Hello @kisak-valve, you are right, I tested with Dota Underlords which is the only other Vulkan-based game I have in my library and same issue after the launch. So I verified with the vulkaninfo command line :
WARNING: [Loader Message] Code 0 : terminator_CreateInstance: Failed to CreateInstance in ICD 4.  Skipping ICD.
Segmentation Fault (core dumped)

So, it doesn't come from Dota 2, I apologize for the inconvenience, you can close the bug.

**Comment by goldspoon123** [comment]
How were you able to resolve the issue? I have same exact problem, both dota 2 and dota underlord crashes, likely due to vulkan. Please tell me how you fixed it

**Comment by darkbaboon** [comment]
Hi @goldspoon123 ! Yes I was able to resolve the issue. As @kisak-valve pointed it, there was something wrong with my vulkan driver. There are three implementations of vulkan for AMD :
- vulkan-radeon - RADV (part of Mesa project)
- amdvlk - AMDVLK Open (maintained by AMD)
- vulkan-amdgpu-pro - AMDVLK Closed (maintained by AMD)

This three drivers can be installed simultaneously. And this is the root cause of my issue.
Since my last system update, the amdvlk driver doesn't work anymore and it was the preferred choice from my linux distribution.
So, I uninstalled the amdvlk driver and the vulkan-radeon became the one used.
Then I relaunched vulkaninfo and there was no more issue and since both Dota 2 and Dota Underlords work again.

Hope it can help you to fix your issue too.
